packages/preact-router/src/lazyRouteComponent.tsx

```
@@ -0,0 +1,90 @@
+import { createElement as h, Fragment } from 'preact'
```

ESLint

[error] Member 'Fragment' of the import declaration should be sorted alphabetically.

(sort-imports)